### PR TITLE
Tech Debt: remove network dependencies

### PR DESCRIPTION
**Pull request by mikesherov**
Goal required 2 types of changes:

1. move plugins into be.net that completely depended on network for core functionality.
2. move bits of code from be.net into this repo where possible and small enough.

This led to some new badness getting into this repo, and some badness will move into the network `assets/js/deprecated` folder, but the greater good of unexpressed cross repo dependencies going away.

### CODE REVIEW

File: be/colorpicker.js [hunk 1]
```diff
@@ -1,12 +1,232 @@
...
   'jquery',
-  'beff/util/cookie',
-  'utils/colors',
   'jqueryui/draggable',
   'jqueryui/slider'
-], function($, cookie, Colors) {
+], function($) {
   'use strict';
 
+  var Colors = {
```
Comment (Attamusc) [inline be/colorpicker.js]: I'm assuming this is just utils/colors inlined, ya?

Comment (mikesherov) [inline be/colorpicker.js]: ding ding

Comment (mikesherov) [inline be/colorpicker.js]: please ignore this inlining completely. It's a Melania Trump job from some code on the network.

File: be/creativefieldspopup.js [hunk 2]
```diff
@@ -1,249 +0,0 @@
-define([
```
Comment (mikesherov) [inline be/creativefieldspopup.js]: this file will move into the network proper. So many network deps.

File: be/previewcrop.js [hunk 3]
```diff
@@ -1,686 +0,0 @@
-// TODO: move this out of here and into a codebase
```
Comment (mikesherov) [inline be/previewcrop.js]: this file will move into the network proper. So many network deps.

File: plugins/jquery.popup.js [hunk 4]
```diff
@@ -1,4 +1,48 @@
 (function ($) {
+  $.fn.clonePosition = function(srcObj, passedParams) {
```
Comment (Attamusc) [inline plugins/jquery.popup.js]: Why does this need to be duped between this and above?

File: be/creativefieldspopup.js (hunk 2 above)
Comment (Attamusc) [inline be/creativefieldspopup.js]: Cool, that's what I figured 👍

**Comment by Attamusc** [comment]
👍

File: plugins/jquery.popup.js (hunk 4 above)
Comment (Attamusc) [inline plugins/jquery.popup.js]: nvm; just read the above response; I imagine it's the same thing

File: be/textbubble.js [hunk 5]
```diff
@@ -1,22 +1,8 @@
...
-    });
-  }
-  else {
-    return jQuery && factory.call(this, jQuery);
-  }
-}(function($, bubblelist) {
+define([
+  'jquery',
+  './bubblelist',
+  'jqueryui/widget'
```
Comment (kaicataldo) [inline be/textbubble.js]: Does this not need to be included as an argument in the callback function?

**Comment by kaicataldo** [comment]
Assuming the above is true,

**Comment by kaicataldo** [comment]
:+1:

File: be/textbubble.js (hunk 5 above)
Comment (mikesherov) [inline be/textbubble.js]: Nah, it's an implicit dep like most jQuery plugins sadly.

File: be/textbubble.js [hunk 6]
```diff
@@ -1,22 +1,8 @@
...
-            './bubblelist',
-            'jqueryui/widget' ], function() {
-      var module = factory.apply(this, arguments);
-      return module;
-    });
-  }
-  else {
-    return jQuery && factory.call(this, jQuery);
-  }
-}(function($, bubblelist) {
```
Comment (mikesherov) [inline be/textbubble.js]: @kaicataldo notice this line!

**Pull request merged**